examples/client/common.c

```diff
@@ -677,6 +687,247 @@ int ClientUseCert(const char* certName, void* heap)
     return ret;
 }
 
+#ifdef WOLFSSH_TPM
+
+#define TPM2_DEMO_STORAGE_KEY_HANDLE    0x81000200  /* Persistent Storage Key Handle (RSA) */
+
+static const char gStorageKeyAuth[] = "ThisIsMyStorageKeyAuth";
```

Comment:
The SRK handle and password being hard coded is not good. Both of these should be macros wrapped with `#ifndef` at a minimum. Ideally they would be arguments to the example client or not needed at all.

If I were writing this example I would create the key using the endorsement primary key... then you would not need anything related to the SRK... It's a pretty easy change, mostly to your README.md to create the key under the endorsement hierarchy using `-eh`.